src/packages/recompose/__tests__/mapPropsStream-test.js

```diff
@@ -1,20 +1,32 @@
 import test from 'ava'
 import React from 'react'
-import { Observable, Subject } from 'rx'
-import { withState, compose, branch } from 'recompose'
-import { mapPropsStream, createEventHandler } from '../'
+import { combineLatest } from 'rxjs/operator/combineLatest'
+import { startWith } from 'rxjs/operator/startWith'
+import { scan } from 'rxjs/operator/scan'
+import { _do } from 'rxjs/operator/do'
+import { map } from 'rxjs/operator/map'
+import {
+  withState,
+  compose,
+  branch,
+  mapPropsStream,
+  createEventHandler
+} from '../'
 import { mount, shallow } from 'enzyme'
 
+import { Observable } from 'rxjs/Observable'
+global.Observable = Observable
+
 const identity = t => t
 
 test('mapPropsStream maps a stream of owner props to a stream of child props', t => {
   const SmartButton = mapPropsStream(props$ => {
     const { handler: onClick, stream: increment$ } = createEventHandler()
     const count$ = increment$
-      .startWith(0)
-      .scan(total => total + 1)
+      ::startWith(0)
```

Comment:
Bind operator is still at stage 0 so syntax may change in future and there are no guarantees it will be included in spec